EF-Adapter/packages.config

```
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Casbin.NET" version="1.2.5" targetFramework="net472" />
+  <package id="DynamicExpresso.Core" version="2.3.1" targetFramework="net472" />
```

Why don't we target .NET Framework 4.5? I think there are still a lot of people using 4.5 right now.

---

```suggestion
  <package id="DynamicExpresso.Core" version="2.3.1" targetFramework="net472" />
```
Then alright.I used latest stable version which installed on my PC.If won't be big issue if they update the their .NET framework version it will be shorted.What should I do now ? Do I need to keep it on 4.7 or 4.5

---

net45 is better.

---

Ok. I'll update it and give a PR.

---

Find the updated PR.